parity/configuration.rs

```diff
@@ -455,8 +457,9 @@ impl Configuration {
 	}
 
 	fn max_peers(&self) -> u32 {
-		let peers = self.args.arg_max_peers as u32;
-		max(self.min_peers(), peers)
+		self.args.arg_max_peers
+			.or(self.args.arg_min_peers)
+			.unwrap_or(DEFAULT_MAX_PEERS) as u32
```

Comment:
In case `max_peers` isn't defined can we set it to `max(min_peers, DEFAULT_MAX_PEERS)`? With the current code if you start parity with just `--min-peers 5`, max peers will also be set to 5 (I'd expect it to use the default value).

Reply:
Yepp, I'll buy that when running ```parity --min-peers 5``` to set set max_value to default